package.json

```diff
@@ -75,12 +75,14 @@
     "joi": "17.13.3",
     "joi-objectid": "4.0.2",
     "lodash": "4.17.21",
+    "lunr": "2.3.9",
     "marked": "14.1.3",
     "marked-alert": "2.1.1",
     "nunjucks": "3.2.4",
     "pino": "9.5.0",
     "pino-pretty": "11.3.0",
     "qs": "6.13.0",
+    "remove-markdown": "0.6.0",
```

Comment:
nitpicking, but there's a plugin for our existing markdown library marked-plainify. wondering if it makes sense to use that and keep all out markdown libs in the same ecosystem?